### PR TITLE
Some minor trigger type fixes

### PR DESCRIPTION
**Pull request by shapirone**
###  Summary

Some minor type specifications for triggers

### Requirements (place an `x` in each `[ ]`)
* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/deno-slack-api/blob/main/.github/CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: src/typed-method-types/workflows/triggers/scheduled.ts [hunk 1]
```diff
@@ -29,9 +29,10 @@ type BaseFrequencyType = {
   /** @description How often the trigger will repeat, respective to the frequency type */
   repeats_every?: number;
   /**
-   * @description The nth week of the chosen frequency type (not available for daily or weekly triggers)
-   * @example The 3rd week of the month */
-  on_week_num?: number;
+   * @description The nth week of the chosen frequency type (not available for daily, weekly, or yearly triggers)
```
Comment (jmplexi) [inline src/typed-method-types/workflows/triggers/scheduled.ts:32]: Not sure if it's entirely clear from the description that "-2" would not be a valid entry. I imagine someone might try to use -2 for the second last week of the month

**Review by jmplexi — COMMENTED** [review]
minor comment about description of on_week_num field

File: src/typed-method-types/workflows/triggers/scheduled.ts (hunk 1 above)
Comment (shapirone) [reply src/typed-method-types/workflows/triggers/scheduled.ts:32]: Yeah, `-2` could be the second to last week in the month, but the options in this union type are specified by the backend. If a developer tries to use `on_week_num: -2` they'll get a type error

File: src/typed-method-types/workflows/triggers/mod.ts [hunk 2]
```diff
@@ -17,11 +17,13 @@ type WorkflowInput = {
   value: any;
 };
 
+type WorkflowStringFormat = `${string}#/workflows/${string}`;
```
Comment (selfcontained) [inline src/typed-method-types/workflows/triggers/mod.ts:20]: wow had no idea you can do that - slick!

**Review by selfcontained — APPROVED** [review]
Nice additions, thanks!

File: src/typed-method-types/workflows/triggers/mod.ts (hunk 2 above)
Comment (shapirone) [reply src/typed-method-types/workflows/triggers/mod.ts:20]: Yeah, [Template Literals](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html) are really cool! I actually saw a tweet about them validating url structures and remembered they existed

**Pull request merged**